src/plugins/intel_cpu/src/nodes/batch_to_space.cpp

```diff
@@ -206,36 +206,36 @@ void BatchToSpace::batchToSpaceKernel() {
             finish[3] = (outShape5D[3] + blockShape[3] - 1 - oAdd[3]) / blockShape[3];
             begin[4] = (blockShape[4] - 1 - oAdd[4]) / blockShape[4];
             finish[4] = (outShape5D[4] + blockShape[4] - 1 - oAdd[4]) / blockShape[4];
-            const int64_t addTmpOC = blocked ? 0LU : oAdd[1];
-            const int64_t addTmpOc = blocked ? oAdd[1] : 0LU;
+            const int64_t addTmpOC = blocked ? 0 : static_cast<int64_t>(oAdd[1]);
```

Comment:
this is already int64_t, isn't it?
Please double check every change that is really necessary after disablement of some data types.

Reply:
Removed